src/test/java/scenario/ScenarioSteps.java

```diff
@@ -183,6 +183,18 @@ public ScenarioSteps() throws IOException {
                     }
                 });
 
+	    Given("I have a gateway as user {word} using the {word} connection profile without a Wallet",
+	            (String userName, String tlsType) -> {
```

Comment:
Some of this code is a duplicate of the `I have a gateway as user {word} using the {word} connection profile` step definition. It would be good to pull out the common parts to another function to avoid duplication.